lib/game.rb

```diff
@@ -1,27 +1,43 @@
 require_relative './player.rb'
 require_relative './board.rb'
 require_relative './ui.rb'
+require 'set'
 
 class Game
   include UI
-  def initialize(board,players)
+  def initialize(board, players)
     @board = board
     @players = players
+  end
+
+  def play
     output('Welcome to Tic-tac-toe!', true)
```

Comment:
Isn't there another way to display this welcome message without it being inside the Game class?

Reply:
Moved it to UI.